app/controllers/audits_controller.rb

```diff
@@ -1,11 +1,14 @@
-class AuditsController < ReactController
+class AuditsController < ApplicationController
   include Foreman::Controller::AutoCompleteSearch
 
   before_action :setup_search_options, :only => :index
 
   def index
     @audits = resource_base_search_and_page.preload(:user)
-    @host = resource_finder(Host.authorized(:view_hosts), params[:host_id]) if params[:host_id]
```

Comment:
does url such as http://localhost:5000/hosts/hostname/audits work as expected after removing this file?

Reply:
works